lib/scanner/index.js

```diff
@@ -81,30 +77,22 @@ class Scanner extends Emitter {
    * @private
    */
   onItemFound(item) {
-    var url = item.url;
+    var shouldPopulate = this.shouldPopulateItem(item.url);
```

Comment:
This can be null, if the option objects doesnt contain the `shouldPopulateItem` this may fail.